.github/workflows/code_health.yml

```diff
@@ -28,4 +28,16 @@ jobs:
       run: npm run test:coverage
 
     - name: Build
-      run: npm run build
\ No newline at end of file
+      run: npm run build
+
+    - name: Check dist directory
+      run: |
+        FILES=$(git status -s dist/)
+        if [ -n "$FILES" ]; then
+          echo "You need to include the changes in the dist folder, execute: npm run build and commit the changes in the dist folder"
+          echo "Uncommitted files:"
+          echo "$FILES"
+          echo "Diff content:"
+          git diff dist/
+          exit 1
+        fi
```

Comment:
[NIT] Missing newline

Reply:
Solved in https://github.com/ossf/scorecard-monitor/pull/84/commits/d21a005ee302e472b039cfc367c4255a92ff91b2